JavaScript/tests/syntax_test_js_bindings.js

```diff
@@ -0,0 +1,38 @@
+// SYNTAX TEST "Packages/JavaScript/JavaScript.sublime-syntax"
+
+const x = value;
+//    ^ meta.binding.name variable.other.readwrite
+
+
+const [ x, y, ...z, ] = value;
+//    ^^^^^^^^^^^ meta.binding.destructuring.sequence
```

Comment:
These assertions should probably continue for another 4 characters, right?

Reply:
Yes.